hack/loadtest/main.go

```diff
@@ -100,14 +100,18 @@ func runLoadTest(ctx context.Context, config *rest.Config) {
 				case <-ctx.Done():
 					return
 				default:
-					if err = util.MeasureOnePlacement(ctx, hubClient, time.Duration(*placementDeadline)*time.Second, time.Duration(*pollInterval)*time.Millisecond, *maxCurrentPlacement, clusterNames, crpFile); err != nil {
-						klog.ErrorS(err, "placement load test finished")
+					loopCtx, cancel := context.WithCancel(context.Background())
+					defer cancel()
```

Comment:
Hi Britania! A nit here: it seems that at this moment this context is only cancelled after all the work is done; if you just would like a placeholder context, a background one would be better.

Reply:
At this moment, since we will create one context in each for loop run, there will be many cancel() calls being deferred, even though the context itself has been cancelled right after the measurement is completed.